server_config.go

```diff
@@ -96,7 +99,8 @@ func (cfg *ServerConfig) GetEnv() (env []string) {
 	return
 }
 
-// makeCommands returns new command provider based on configured options.
+//=================================== PRIVATE METHODS ======================================================
```

Comment:
? new comment style?

Reply:
Just to separate private and public methods